src/internal/storage/chunk/client.go

```diff
@@ -79,7 +79,7 @@ func (c *trackedClient) beforeUpload(ctx context.Context, chunkID ID, md Metadat
 		pointsTo = append(pointsTo, cid.TrackerID())
 	}
 	chunkTID := chunkID.TrackerID()
-	if err := dbutil.WithTx(ctx, c.db, func(tx *pachsql.Tx) (retErr error) {
+	if err := dbutil.WithTx(ctx, c.db, func(ctx context.Context, tx *pachsql.Tx) (retErr error) {
```

Comment:
So since we are passing in a new `ctx`, below we should use the Contexted version of the sqlx API, like `tx.SelectContext()` etc.